neuralmonkey/readers/plain_text_reader.py

```diff
@@ -19,6 +23,38 @@ def reader(files: List[str]) -> Iterable[List[str]]:
     return reader
 
 
+def column_separated_reader(column: int, delimiter: str = "\t",
+                            quotechar: str = None, encoding: str = "utf-8"):
+    """Get reader for delimiter-separated tokenized text.
+    
+    Args:
+        column: number of column to be returned. It starts with 1 for the first
+    """
+    def reader(files: List[str]) -> Iterable[List[str]]:
+        text_reader = get_plain_text_reader(encoding)
+        for line in text_reader(files):
+            f = io.StringIO(' '.join(line))
+            parsed_csv = list(csv.reader(f, delimiter=delimiter,
+                                         quotechar=quotechar,
+                                         skipinitialspace=True))
+            if len(parsed_csv) != 1:
```

Comment:
Kontroluješ, jestli to neni větší nebo jestli to neni nula? Nebo to má bejt jednička vždycky? Pak ale nechápu tu hlášku..